virttest/qemu_vm.py

```diff
@@ -2281,8 +2281,12 @@ def sort_key(dev):
             attr_info = ["timestamp", params["msg_timestamp"], bool]
             add_qemu_option(devices, "msg", [attr_info])
         if params.get("realtime_mlock"):
-            attr_info = ["mlock", params["realtime_mlock"], bool]
-            add_qemu_option(devices, "realtime", [attr_info])
+            if devices.has_option("overcommit"):
```

Comment:
@huangyum  is this command works for ppc platform?

Reply:
@xutian Yes, it works same as x86_64.